src/Frame.cc

```diff
@@ -1397,6 +1397,46 @@ Frame::recalcResizeDrag(int nx, int ny, bool left, bool top)
 	_gm.y = top ? (_click_y - _gm.height) : _click_y;
 }
 
+void
+Frame::moveToHead(const std::string& arg)
+{
+	char* endp = NULL;
+	uint head_nr = strtol(arg.c_str(), &endp, 10);
+	if (endp && *endp == '\0') {
+		moveToHead(head_nr);
+		return;
+	}
+	if (StringUtil::ascii_ncase_equal(arg, "LEFT")) {
```

Comment:
I wold much rather have this as an if else if else where the new head number is calculated, and at the end do if (head_nr == -1) { P_ERR() } else { moveToHead(head_nr); } getting rid of all the returns and moveToHead calls.